charts/prometheus-conntrack-stats-exporter/values.yaml

```
@@ -42,7 +42,11 @@ resources: {}
   #   memory: 15Mi
 
 securityContext:
-  privileged: true
+  runAsUser: 1000
+  runAsNonRoot: true
+  readOnlyRootFilesystem: true
+  allowPrivilegeEscalation: false
+  privileged: false
```

the container does not work without privileged permissions.

This would work though:

```suggestion
      capabilities:
        drop:
          - ALL
      privileged: true
      readOnlyRootFilesystem: true
      seccompProfile:
        type: RuntimeDefault

```